stylesheets/atom-lint.less

```diff
@@ -186,7 +186,7 @@
   }
 }
 
-.editor .gutter .line-number {
+.atom-text-editor .gutter .line-number {
```

Comment:
Here and following the suggestion Atom makes, it would be better to have ".atom-overlay" it would not only get rid of the deprecated ".editor" it would also avoid deprecated calls